student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] Like the previous reviewer mentioned, note that as this class inherits from Person class its constructor should extend all parent's instance variables so kindly add the rest of the variables to initialize
``` def initialize(age, classroom, name = "Unknown", parent_permission = true)  ```